formats/properties/commonMain/src/kotlinx/serialization/Properties.kt

```diff
@@ -62,6 +62,10 @@ public class Properties(override val serializersModule: SerializersModule = Empt
         override fun encodeTaggedNull(tag: String) {
             // ignore nulls in output
         }
+
+        override fun encodeTaggedEnum(tag: String, enumDescription: SerialDescriptor, ordinal: Int) {
```

Comment:
It looks like parameter name should be `enumDescriptor` instead of `enumDescription`. Seems like an oversight in `TaggedEncoder`, pls rename it, if you don't mind